Readme.md

```diff
@@ -1,16 +1,85 @@
+  <a href="#downloading">Downloading</a> •
+  <a href="#developers">Developers</a> •
+  <a href="#wiki">Wiki</a> •
+  <a href="#contributing">Contributing</a> •
+  <a href="#issues">Issues</a> •
+  <a href="#support">Support</a> •
+  <a href="#license">License</a>
+</p>
+
+---
+
+## About
+
+MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Android platform. It allows you to:
+
+* View release information by scanning a barcode
+* Search for information about artists, releases, release groups,labels, recordings, instruments, and events
+* View collections
+* Tag audio files like Picard
+* Donate to the MetaBrainz Foundation via PayPal
+
+## Installation
+	    
+* First start with installing Android Studio on your pc.
+* Clone this repository.
+* Use the `gradlew build` command to build the project directly or use the IDE to run the project to your phone or the emulator.
+
+## Downloading
+
+* [Google Play Store](https://play.google.com/store/apps/details?id=org.metabrainz.android)
+	    
+* [F-Droid](https://f-droid.org/en/packages/org.metabrainz.android/)
+	 
+## Developers
+	    
+MusicBrainz App was first created in 2010-11 as a part of GSoC by Jamie McDonald. The project went stagnant for a few years.
 
 In GSoC 2019, development on the project started again and was later taken up as another GSoC project by amCap1712.
 
-In GSoC 2020,combined effort of SomalRudra & amCap1712 revamped the app design, improved overall infrastructure, integrated testing,Continous Integration & has been successful to implement the key feature:Tagger
+In GSoC 2020,combined effort of SomalRudra & amCap1712 revamped the app design, improved overall infrastructure, integrated testing,Continous Integration & has been successful to implement the key feature : Tagger
+
+## Wiki
+
+Do you **need some help**? Check the _articles_ from the wiki.
+
+## Contributing
+
+Got **something interesting** you'd like to **share**? Learn about contributing.
+	    
+## Issues
+	  
+If you think you have found a bug, please report it on the [issue tracker](https://tickets.metabrainz.org/projects/MOBILE/issues). The app is under active development again since 2019 and some new features are planned. You can suggest and vote for new features in the same location.
+	    
+## Support
+
+Reach out to me at one of the following places:
+
+- Website at [musicbrainz.org](https://musicbrainz.org)
+- E-Mail: **support@metabrainz.org**
 
-# What's next
+## License
 
-* The tagger is still in initial stages. The app needs to achieve complete usability with the desktop version of Picard. This requires that we support all the tags as Picard.
-* Moving a larger part of the codebase to Kotlin.
-* Incorporate functionality from more *Brainz.
+This Project is licensed under the [GPL version 3 or later](https://www.gnu.org/licenses/gpl-3.0.html) with sections under the [Apache License version](https://www.apache.org/licenses/LICENSE-2.0.html) 
```

Comment:
I don't remember offhand what parts under Apache license. Which files did you find?

Reply:
I found this data at musicbrainz.org/doc/MusicBrainz_for_Android